__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const db = require('../db/connection.js')
+const request = require('supertest')
+const seed = require('../db/seeds/seed.js')
+const data = require('../db/data/test-data/index.js')
+const app = require('../app.js')
+
+beforeEach(() => seed(data));
+afterAll(() => db.end())
+
+describe('/api/topics', () => {
+    test('GET 200: should return an array of all topics objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            const rows = response.body.rows
+            expect(rows).toHaveLength(3)
+            rows.forEach((row) => {
+                expect(row).toMatchObject({
+                    slug : expect.any(String),
+                    description : expect.any(String)
+                })
+            })
```

Comment:
The response object should have a descriptive key which gives us access to the data we want, instead of having a key of rows something like topics might be more meaningful.